### PR TITLE
Add subnet to deployer output and adjust upload script

### PR DESCRIPTION
**Pull request by rukawata**
## Problem
1. management subnet information will be required for sap_landscape deployment
1. upload script should only be run during first deployment

## Solution
1. Add management subnet to output for `bootstrap/sap_deployer` and `run/sap_deployer`
1. Move `script.tf` and related files to `bootstrap/sap_deployer`
1. Change file permission of post_deployment.sh which will be consistent with sap_lib
1. Change file name to `deployer.terraform.tfstate` to be consistent with sap_lib
1. Add missing StrictHostKeyChecking to `deployer_scp.tmpl`

## Test
1. `cd sap-hana/deploy/terraform/bootstrap/sap_deployer`
1. `terraform init`
1. `terraform apply -auto-approve -var-file=deployer.json`

### CODE REVIEW

File: deploy/terraform/bootstrap/sap_deployer/deployer_scp.tmpl [hunk 1]
```diff
@@ -1,7 +1,7 @@
 %{~ for index, deployer in deployers ~}
-ssh -i ${deployer-ppk} ${deployer.authentication.username}@${deployer-ips[index]} "mkdir -p ~/.deployer/"
+ssh -i ${deployer-ppk}  -o StrictHostKeyChecking=no ${deployer.authentication.username}@${deployer-ips[index]} "mkdir -p ~/.deployer/"
 %{ endfor ~}
 
 %{~ for index, deployer in deployers ~}
-scp -i ${deployer-ppk} -o StrictHostKeyChecking=no ../terraform.tfstate ${deployer.authentication.username}@${deployer-ips[index]}:~/.deployer/deployer_terraform.tfstate
+scp -i ${deployer-ppk} -o StrictHostKeyChecking=no ../terraform.tfstate ${deployer.authentication.username}@${deployer-ips[index]}:~/.deployer/deployer.terraform.tfstate
```
Comment (YunziZhangAzure) [inline deploy/terraform/bootstrap/sap_deployer/deployer_scp.tmpl:6]: After trying terraform init by using file named "deployer.terraform.tfstate", I found it doesn't work unfortunately. I guess we have to use default name as "terraform.tfstate"

Comment (YunziZhangAzure) [inline deploy/terraform/bootstrap/sap_deployer/deployer_scp.tmpl:6]:
```suggestion
scp -i ${deployer-ppk} -o StrictHostKeyChecking=no ../terraform.tfstate ${deployer.authentication.username}@${deployer-ips[index]}:~/.deployer/terraform.tfstate
```

Comment (rukawata) [reply deploy/terraform/bootstrap/sap_deployer/deployer_scp.tmpl:6]: In the backend init, you can specify the key which is the filename (eg. `deployer.terraform.tfstate` in this case).
If we only use terraform.tfstate, it does not scale when we deal with multiple sap landscapes/systems.

Comment (rukawata) [reply deploy/terraform/bootstrap/sap_deployer/deployer_scp.tmpl]: As discussed, I added code in this PR to create .config as well upload deployer.terraform.tfstate to .config :)

**Pull request merged**